system/core/base.php

```diff
@@ -310,7 +273,8 @@ function hypha_setTitle($string) {
 		returns hypha base html file contents
 	*/
 	function hypha_getHtml() {
-		return Hypha::$data->html->read();
+		global $O_O;
+		return $O_O->data->html->read();
```

Comment:
This should be `themeHtml` in stead of `html`; `$O_O->data->themeHtml->read();`

Reply:
Fixed.